draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -350,9 +350,25 @@ control algorithms have the opportunity to improve the state of the art.
 
 ### Fairness within the Alternate Congestion Control Algorithm.
 
-When multiple competing flows all using the same
+When multiple competing flows all use the same
 alternate congestion control algorithm, the proposal should
 explore how the capacity is shared among the competing flows.
+Capacity fairness can be important when a small number of similar
+flows compete to fill a bottleneck. It can however also not be useful:
+for example when comparing flows seek to send at different rates or
+when some of the flows do not last sufficiently long to approach
+asymptotic behavior.
+
+In other contexts, and when differing congestion control
+algorithms are used, it can be valuable to understand whether
+an alternate congestion control algorithm can induce more
+harm to sharing flows than existing
+defined methods. The measure of harm is not restricted to the equality
+of capacity, but ought also to consider metrics such as the
+latency introduced, or an increase in packet loss. This evaluation must
+assess the potential to cause starvation, including assurance that
+a loss of all feedback (e.g., detected by expiry of a retransmission time out)
+results in backoff.
 
```

Comment:
Again, not sure we should differentiate "different algorithms" and "same algorithm"